Cython/Build/Dependencies.py

```diff
@@ -863,8 +863,17 @@ def copy_to_build_dir(filepath, root=root):
                 if np_pythran:
                     c_file = base + '.cpp'
                     options = pythran_options
-                    m.include_dirs.append(pythran_include_dir)
-                    m.extra_compile_args.extend(('-std=c++11','-DENABLE_PYTHON_MODULE','-D__PYTHRAN__'))
+                    pythran_ext = pythran.config.make_extension()
+                    m.include_dirs.extend(pythran_ext['include_dirs'])
+                    m.extra_compile_args.extend(pythran_ext['extra_compile_args'])
+                    m.extra_link_args.extend(pythran_ext['extra_link_args'])
+                    m.define_macros.extend(pythran_ext['define_macros'])
+                    m.undef_macros.extend(pythran_ext['undef_macros'])
+                    m.library_dirs.extend(pythran_ext['library_dirs'])
+                    m.libraries.extend(pythran_ext['libraries'])
+                    # These options are not compatible with the way normal Cython plugins work
```

Comment:
@aguinet there's no guarantee these two options will be inserted by pythran (the user can customize these settings) so you should wrap each ``remove`` in a try block